template.rb

```diff
@@ -167,6 +168,41 @@ def setup_timezone
   gsub_file("config/application.rb", "Central Time (US & Canada)", "Adelaide")
 end
 
+# By default in production Rails is using TaggedLogging
+# replace it with StructuredLogging(with json formatter)
+# and add StructuredLogging(with basic formatter) to development and test
+# Note: should be invoked before #setup_staging
+def setup_logger
+  copy_file("lib/loggers/structured_logging.rb")
+
+  insert_into_file("config/application.rb", after: "Bundler.require(*Rails.groups)\n") do
+    "require_relative \"../lib/loggers/structured_logging\"\n"
```

Comment:
Rails 7.1 introduced autoloading for lib files. I don't think we should be relying on `require_relative` for lib files unless there's a need to do so, in which case that directory should be added to the list of directories that are excluded from autoloading.

Reply:
`require "Koi/loggers/structured_logging"`